### PR TITLE
Solana token transfer tutorial 

### PR DESCRIPTION
**Pull request by aditya-passbird**
- [x]  Token transfer from Developer Wallet.
- [x]  Token Transfer from end-user wallet.

### CODE REVIEW

File: solana-chain-tutorial/scripts/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Your MetaKeep API key
+API_KEY =
```
Comment (rohan-passbird) [inline solana-chain-tutorial/scripts/.env:2]: rename folder to `solana`

File: solana-chain-tutorial/frontend/token-transfer-tutorial/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
```
Comment (rohan-passbird) [inline solana-chain-tutorial/frontend/token-transfer-tutorial/.gitignore:3]: you can remove the `solana` folder inside frontend

File: solana-chain-tutorial/scripts/.env (hunk 1 above)
Comment (aditya-passbird) [reply solana-chain-tutorial/scripts/.env]: done

**Review by rohan-passbird — APPROVED** [review]
👍

**Pull request merged**